articles/quickstart/native/ios-objc/_includes/_login_centralized.md

```diff
@@ -114,6 +114,27 @@ To learn more about the `credentials` object, read the [Credentials](https://git
 
 <%= include('../../../../_includes/_logout_url') %>
 
+To clear on server side the session add the following snipped to the HybridAuth.swift file:
+
+```swift
+// HybridAuth.swift
+
+    @objc
+    func logOutUser(callback: @escaping(Bool, String) -> Void){
+        Auth0
+            .webAuth()
+            .clearSession(federated: true){
```

Comment:
I don't think this parameter is necessary. Can you remove it please?

Reply:
If it's required please set to false, which is what we are doing across all QS @benlizar (default behavior).